src/components/RichText/RichText.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import { Editor } from '@tinymce/tinymce-react'
+
+import 'tinymce/tinymce'
+
+// Basic tinyMCE theme & skins required for editor to display
+import 'tinymce/themes/silver/theme.min'
+import 'tinymce/skins/ui/oxide/skin.min.css'
+import 'tinymce/skins/ui/oxide/content.min.css'
+
+// Import required plugins
+import 'tinymce/plugins/autolink/plugin.min'
+import 'tinymce/plugins/lists/plugin.min'
+import 'tinymce/plugins/link/plugin.min'
+import 'tinymce/plugins/table/plugin.min'
+import 'tinymce/plugins/paste/plugin.min'
+import 'tinymce/plugins/charmap/plugin.min'
+
+interface Props {
+  // id of the rich text editor component
+  id?: string
+  // Initial value of the rich text editor
+  value?: string
+  // Defines whether the rich text editor should be enabled/disabled (default = false)
+  disabled?: boolean
+  // Height of the rich text editor
+  height?: number
+  // Method run on the editors onEditorChange event. Returns editor content as HTML.
+  onChange?: (event: React.KeyboardEvent<HTMLTextAreaElement>) => void
```

Comment:
Unfortunately this props comments are not catched up by the storybook docgen. Could you please adds description and defaults for your props in the following way?
```
/**
   * Defines the color of the alert. Defaults to primary.
   * @default "primary"
   */
```
You should then see the prop table filling up in the component's storybook page. You can take a look at other components prop description to get some hints (like the Alert component, from which I took the upper code).